### ISSUE TITLE
Windows: Install SCION as services

### ISSUE THREAD

**Issue by lschulz** [netsys-lab/scion-orchestrator#14]
SCION should be installed as services on Windows.

**Comment by martenwallewein** [comment]
Yes, I already started adapting your changes to SCION for the orchestrator, current state is here, will need some time to complete it: https://github.com/netsys-lab/scion-orchestrator/tree/feature/windows-service-executables

**Comment by lschulz** [comment]
Great, let me know if I can help with anything.
